### PR TITLE
tree alignment

### PR DESCRIPTION
**Pull request by dkalathil**

### CODE REVIEW

File: src/components/DataStructures/Graph/GraphTracer.js [hunk 1]
```diff
@@ -379,26 +379,32 @@ class GraphTracer extends Tracer {
...
       const linkedNodes = this.findLinkedNodes(node.id, false);
       if (sorted) linkedNodes.sort((a, b) => a.id - b.id);
       for (const linkedNode of linkedNodes) {
         if (marked[linkedNode.id]) continue;
-        recursivePosition(linkedNode, h, v + 1);
+        let x1 = x;
+        let y1 = y;
+        // For left child
+        if (linkedNode.id === 2 * node.id) {
+          x1 -= leafCounts[node.id] * 30;
```
Comment (alextroup-uom) [inline src/components/DataStructures/Graph/GraphTracer.js]: could this be commented? I'm not sure what the numbers mean here

Comment (dkalathil) [reply src/components/DataStructures/Graph/GraphTracer.js]: Actually we can't ..  For each leafnodes we are assigning 30. If we remove that the tree will become vertical.

Comment (alextroup-uom) [reply src/components/DataStructures/Graph/GraphTracer.js]: Ah - not commented out, commented as in explained - or the numbers could be factored out into const declarations with descriptive names?

Comment (dkalathil) [reply src/components/DataStructures/Graph/GraphTracer.js]: Added constants and comments regarding two variables

**Pull request merged**